README.md

```diff
@@ -227,6 +233,7 @@ This software is released under the MIT license. See [the license file](LICENSE)
 details.
 
 [videojs-docs]: http://docs.videojs.com/tutorial-plugins.html
+[videojs-translation]: http://docs.videojs.com/tutorial-languages.html
```

Comment:
I don't see where this is used ... is the link above to `videojs-docs` incorrect? If so, I think the other PRs may be incorrect.

Reply:
Yes, it was meant to be a link to videojs-translation.